packages/components/aside/src/aside.css

```diff
@@ -0,0 +1,144 @@
+/* General........................................................ */
+
+.aside-container {
+	position: absolute;
+	top: 0;
+	bottom: 0;
+	right: 0;
+	left: auto;
+	display: flex;
+	flex-direction: column;
+	min-height: 100vh;
+	width: var(--ts-aside);
+	background-color: var(--ts-color-white);
+	box-shadow: 0 0 3px var(--ts-shadow-color);
+	z-index: var(--ts-zindex-aside);
+}
+
+.aside-container > * {
+	padding: var(--ts-unit);
+}
+
+/* Header......................................................... */
+
+.aside-container header {
+	height: var(--ts-unit-triple);
+	background-color: var(--ts-color-white);
+	display: flex;
+	border-bottom: 1px solid var(--ts-border-color-default);
+}
+
+.aside-container .aside-title {
```

Comment:
I think these `.aside-container` selectors are redundant, and can be dropped to have cleaner selectors

Reply:
But not sure, do what you think is better!